documentation/Tophatting documentation.md

```diff
@@ -6,20 +6,24 @@ When components are updated in ways that impact how people use them, their `READ
 
 When you update the `README.md` of a component, that content can change the documentation that appears in the [Polaris style guide](https://polaris.shopify.com/).
 
-Manually testing documentation (we call it “[tophatting](https://github.com/Shopify/polaris-react/blob/main/documentation/Tophatting.md#history-of-the-phrase)”, or 🎩 for short) adds that extra degree of certainty that what you’re shipping looks like what you expect.
+Manually testing documentation (we call it “[tophatting](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#history-of-the-phrase)”, or 🎩 for short) adds that extra degree of certainty that what you’re shipping looks like what you expect.
 
 ## When should you 🎩 documentation?
 
-When you’re adding content to a component’s `README.md` (such as `/src/components/Button/README.md`) to change or add best practices, accessibility documentation, or other information.
+When you’re adding content to a component’s `README.md` (such as `polaris-react/src/components/Button/README.md`) to change or add best practices, accessibility documentation, or other information.
 
 ## What should you 🎩?
 
 Check the location, appearance, and readability of the content you add to the style guide.
 
+<!-- To be replaced by monorepo method soon
+
 ## How to 🎩 documentation
 
-1. Check out the `master` branch from [`polaris-styleguide`](https://github.com/Shopify/polaris-styleguide)
+1. Check out the `main` branch from [`polaris-styleguide`](https://github.com/Shopify/polaris-styleguide)
```

Comment:
What do you think we should do with this section for now?? Related to the changes and recommendations [in our `CONTRIBUTING.md`](https://github.com/Shopify/polaris/pull/5523#discussion_r849817750)